third_party/tensorrt/local/BUILD

```diff
@@ -67,11 +67,6 @@ cc_import(
         ":windows": "lib/nvinfer.dll",
         "//conditions:default": "lib/x86_64-linux-gnu/libnvinfer.so",
     }),
-    static_library = select({
-        ":aarch64_linux": "lib/aarch64-linux-gnu/libnvinfer_static.a",
-        ":windows": "lib/nvinfer.lib",
-        "//conditions:default": "lib/x86_64-linux-gnu/libnvinfer_static.a"
```

Comment:
Do you know if there is a solution here other than removing, the issue i see is a symbol issue for static

Reply:
At the moment, I don't have any workaround or fix for this. If I skip the aarch64_linux. It will default to x86_64

Reply:
Separated static library target